app/user_viewmembers_test.go

```diff
@@ -1101,3 +1108,31 @@ func TestResctrictedViewMembers(t *testing.T) {
 		}
 	})
 }
+
+func setupWebSocketConnection(th *TestHelper, t *testing.T, userIDs ...string) func() {
```

Comment:
Probably this function should go in the `app/helper_test.go` file, and maybe as a method of TestHelper struct.